bc_obps/registration/tests/models/test_operation.py

```diff
@@ -192,3 +194,40 @@ def test_unique_swrs_facility_id_constraint(self):
 
         with self.assertRaises(ValidationError, msg="Operation with this Swrs facility id already exists."):
             invalid_operation.save()
+
+    # BCGHG ID generation tests
+
+    def test_cannot_create_operation_with_duplicate_bcghg_id(self):
+        bcghg_id_instance = baker.make(BcGreenhouseGasId, id='14121100001')
+        operation_instance: Operation = baker.make_recipe('utils.operation')
+        operation_instance.bcghg_id = bcghg_id_instance
+        operation_instance.save(update_fields=['bcghg_id'])
+        with pytest.raises(ValidationError, match='Operation with this Bcghg id already exists.'):
```

Comment:
Adding a note cause this confused me for awhile--the OneToOneField appears to automatically set up a unique constraint. I didn't have to additionally add one